src/Services/CustomEmailSender.cs

```diff
@@ -15,7 +17,32 @@ public Task SendResetPasswordTokenAsync(string email, string callbackUrl)
             string subject = "Sotex Staff hub - Your account is created";
             string message = $"Please set your password by <a href='{HtmlEncoder.Default.Encode(callbackUrl)}'>clicking here</a>.";
 
-            //here will be SendGrid logic
+            try
+            {
+                var smtpClient = new SmtpClient("smtp.gmail.com")
+                {
+                    Port = 587,
+                    Credentials = new NetworkCredential("nikola.j00@gmail.com", "GMAIL_PASS"),
+                    EnableSsl = true,
+                };
+
+                var mailMessage = new MailMessage
+                {
+                    From = new MailAddress("nikola.j00@gmail.com"),
+                    Subject = subject,
+                    Body = message,
+                    IsBodyHtml = true,
+                };
+
+                mailMessage.To.Add(email);
+
+                smtpClient.Send(mailMessage);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Error sending email: {ex.Message}");
+            }
+
```

Comment:
Don't push personal info as code, its a security issue and usually you will be asked to rebase. Usually for sending any kinds of notifications will require a mailing server. This code uses only googles mail server and you probably don't want to depend on that. What we need is an interface to abstract this logic and then one implementation can be an smtp client, but other would be a push to some kind of localhost mail servers. Mailhog is a good example of such server. 
*NOTE*: Don't ever push your credentials (username is also a credential). Either read them from a .gitignored file (like .env files) or from environment variables. Thanks